snuba/datasets/discover.py

```diff
@@ -154,6 +154,9 @@ def __init__(self) -> None:
                 ("username", Nullable(String())),
                 ("email", Nullable(String())),
                 ("ip_address", Nullable(String())),
+                # SDK
+                ("sdk_name", Nullable(String())),
+                ("sdk_version", Nullable(String())),
```

Comment:
What does sdk_integrations mean for a transaction? Do we need to handle this in some way?

Reply:
`sdk_integrations` isn't stored at all for transactions. I talked briefly with @untitaker and @HazAT this morning and they see a need to make it searchable in the short term. If we want it for correctness then we'll need more storage columns or to put it in tags/contexts.